test/evergreen.yml

```diff
@@ -358,6 +358,17 @@ functions:
           cd build_posix
           ${test_env_vars|} ${make_command|make} VERBOSE=1 check ${smp_command|} 2>&1
         fi
+
+  "cppsuite test":
+   - command: shell.exec
+     params:
+       working_dir: "wiredtiger/build_posix/"
```

Comment:
Can the `/` at the end of the string be removed to be consistent with other tasks?

Reply:
Fixed in 7a7c6f5532fc434d145f29ddc7aad936f9da3dcc